data/sql/updates/pending_db_world/rev_1637592933273396000.sql

```diff
@@ -0,0 +1,3216 @@
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMPLATE+10, 0, 'Copper Vein'),
+(@FREE_GAMEOBJECT := @FREE_GAMEOBJECT+1, @FREE_POOL_TEMP
```

Comment:
sql in those lines doesn't seem to comply to standards
1) missing backticks around table name
2) no need to specify table name along the field names
```suggestion
UPDATE `pool_gameobject` SET `chance` = 70 WHERE `description` LIKE 'Ghostlands Spawn Point%(Tin)%';
```